src/docdb/tree/DocDBDocumentsTreeItem.ts

```diff
@@ -57,39 +58,33 @@ export class DocDBDocumentsTreeItem extends DocDBTreeItemBase<RetrievedDocument>
 
         if (docID || docID === "") {
             docID = docID.trim();
-            let body = { id: docID };
-            body = <NewDocument>(await this.promptForPartitionKey(body));
+            let body: ItemDefinition = { id: docID };
+            body = (await this.promptForPartitionKey(body));
             context.showCreatingTreeItem(docID);
-            const document = await this.createDocument(body);
+            const item: ItemResponse<ItemDefinition> = await this.root.getDocumentClient().database(this.parent.parent.id).container(this.parent.id).items.create(body);
```

Comment:
These lines and `this.createDocument` just below it are duplicated. Either remove `createDocument` or call it here